### PR TITLE
#1016 - Add faster zonotope interval box approximation

### LINKED ISSUE #1016: Method to overapproximate a zonotope with an axis-aligned box
A zonotope can be approximated with an axis-aligned box using the method in [1]. This box is actually tight. One can actually do `LazySets.Approximations.overapproximate(Z, Hyperrectangle)` which falls back to `LazySets.Approximations.box_approximation` that evaluates 2n support functions. But using the method in [1] is faster because consists on an element-wise absolute value and a sum over columns; it doesn't need to evaluate support functions.

[1] Althoff, M., Stursberg, O., & Buss, M. (2010). Computing reachable sets of hybrid systems using a combination of zonotopes and polytopes. Nonlinear analysis: hybrid systems, 4(2), 233-249.

### PR DESCRIPTION
**Pull request by mforets**
Closes #1016.

### CODE REVIEW

File: src/Approximations/overapproximate.jl [hunk 1]
```diff
@@ -133,6 +133,37 @@ function overapproximate(S::ConvexHull{N, Zonotope{N}, Zonotope{N}},
...
+
+### Output
+
+A hyperrectangle.
+
+### Algorithm
+
+This function implements the method in [Section 5.1.2, 1]. A zonotope
+``Z = ⟨c, G⟩`` can be overapproximated tightly by an axis-aligned box
+(i.e. an `Hyperrectangle`) such that its center is ``c`` and the radius along
```
Comment (schillic) [inline src/Approximations/overapproximate.jl]:
```suggestion
(i.e. a `Hyperrectangle`) such that its center is ``c`` and the radius along
```

**Comment by mforets** [comment]
Thanks for the review!

Waiting for CI.

File: src/Approximations/overapproximate.jl [hunk 2]
```diff
@@ -133,6 +133,37 @@ function overapproximate(S::ConvexHull{N, Zonotope{N}, Zonotope{N}},
...
+(i.e. a `Hyperrectangle`) such that its center is ``c`` and the radius along
+dimension ``i`` is the column-sum of the absolute values of the ``i``-th row
+of ``G`` for ``i = 1,…, p``, where ``p`` is the number of generators of ``Z``.
+
+[1] *Althoff, M., Stursberg, O., & Buss, M. (2010). Computing reachable sets of
+hybrid systems using a combination of zonotopes and polytopes. Nonlinear analysis:
+hybrid systems, 4(2), 233-249.*
+"""
+function overapproximate(Z::Zonotope, ::Type{<:Hyperrectangle})::Hyperrectangle
+    r = sum(abs.(Z.generators), dims=2)[:]
```
Comment (mforets) [inline src/Approximations/overapproximate.jl]:
```suggestion
    r = Compat.sum(abs.(Z.generators), dims=2)[:]
```
```suggestion
    r = Compat.sum(abs.(Z.generators), dims=2)[:]
```

**Comment by mforets** [comment]
lights are green, thanks for the review ;)

**Pull request merged**